slave_monitor.go

```diff
@@ -15,13 +15,23 @@ type (
 	}
 
 	statistics struct {
+		Processes             float64 `json:"processes"`
+		Threads               float64 `json:"threads"`
+
 		CpusLimit             float64 `json:"cpus_limit"`
 		CpusSystemTimeSecs    float64 `json:"cpus_system_time_secs"`
 		CpusUserTimeSecs      float64 `json:"cpus_user_time_secs"`
 		CpusThrottledTimeSecs float64 `json:"cpus_throttled_time_secs"`
+		CpusNrPeriods         float64 `json:"cpus_nr_periods"`
+		CpusNrThrottled       float64 `json:"cpus_nr_throttled"`
 
 		MemLimitBytes float64 `json:"mem_limit_bytes"`
 		MemRssBytes   float64 `json:"mem_rss_bytes"`
+		MemTotalBytes float64 `json:"mem_total_bytes"`
+		MemCacheBytes float64 `json:"mem_cache_bytes"`
+
```

Comment:
When already adding these metrics, shouldn't this also include `mem_swap_bytes`?